Assets/Scripts/Other/LevelController.cs

```diff
@@ -7,6 +7,8 @@
 public class LevelController : MonoBehaviour
 {
 	[SerializeField] private LevelRound [] levelRounds;
+	[SerializeField] private GameObject playerObject;
```

Comment:
Nitpick, i'd probably rename this to 'PlayerSpawner'

Reply:
This is the player object itself not where it should be placed/will be spawned

Reply:
I got confused as to how the code worked, since it appears that the 'PlayerSpawner' is just a point in space.